From: John Wright <jsw@debian.org>
Date: Fri, 30 Dec 2011 14:57:36 -0700
Subject: Remove needless linking

Upstream's makefile links against libbz2 and libdl, which makedumpfile
doesn't actually use any symbols from.

Signed-off-by: John Wright <jsw@debian.org>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index f7373d1..15c907e 100644
--- a/Makefile
+++ b/Makefile
@@ -38,7 +38,7 @@ $(OBJ_ARCH): $(SRC_ARCH)
 	$(CC) $(CFLAGS_ARCH) -c -o ./$@ ./$(@:.o=.c) 
 
 makedumpfile: $(SRC) $(OBJ_PART) $(OBJ_ARCH)
-	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJ_PART) $(OBJ_ARCH) -o $@ $< -ldw -lbz2 -ldl -lelf -lz
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJ_PART) $(OBJ_ARCH) -o $@ $< -ldw -lelf -lz
 	echo .TH MAKEDUMPFILE 8 \"$(DATE)\" \"makedumpfile v$(VERSION)\" \"Linux System Administrator\'s Manual\" > temp.8
 	grep -v "^.TH MAKEDUMPFILE 8" makedumpfile.8 >> temp.8
 	mv temp.8 makedumpfile.8
-- 
